img/media-types.dot

```diff
@@ -6,6 +6,7 @@ digraph G {
     layer [shape=note, label="Layer tar archive\napplication/vnd.oci.image.layer.v1.tar\napplication/vnd.oci.image.layer.v1.tar+gzip\napplication/vnd.oci.image.layer.nondistributable.v1.tar\napplication/vnd.oci.image.layer.nondistributable.v1.tar+gzip"]
   }
 
+  imageIndex -> imageIndex [label="1..*"]
```

Comment:
If we're going to draw this loop in here, I think we also want to SHOULD support for indexes [here][1] (for both providers and consumers), at least for the index in `index.json`.

[1]: https://github.com/opencontainers/image-spec/blame/v1.0.0/image-index.md#L30